identity/did.go

```diff
@@ -198,10 +199,10 @@ type ServiceDID interface {
 	GetKey(did DID, key [32]byte) (*KeyResponse, error)
 
 	// RawExecute calls the execute method on the identity contract
-	RawExecute(ctx context.Context, to common.Address, data []byte) error
+	RawExecute(ctx context.Context, to common.Address, data []byte) (utxID uuid.UUID, done chan bool, err error)
```

Comment:
Cyclic dependency madness with transactions package. Only way could think about fixing it fast and dirty. Explanation in the implementation.

Reply:
There is a fast and clean way to do it as done by other packages already, just define an interface that TxID automatically implements in this file and return a pointer to TxID type in the ideth package :) 
```
type IDTX interface {
	String() string
	Bytes() []byte
}
```
Thanks!